website/config/bootstrap.js

```diff
@@ -71,6 +76,10 @@ module.exports.bootstrap = async function() {
     password: await sails.helpers.passwords.hashPassword('abc123')
   }).fetch();
 
+  await Platform.create({
+    currentUnfrozenGitHubPrNumbers: []
+  });
```

Comment:
this will expose us to a rare (and probably not really important, except in the spirit of keeping a well-tended garden) concurrency bug, as-is.

A solution could be to use a unique attr to serve as a lock, and lift this create up topside, negotiating its uniqueness constraint error using `.intercept()`. 

Another option is to delete the create altogether and leave a comment explaining why you chose to make it less "magic"/convenient in this case in order to avoid complexity.